dashboard/app/lib/javascripts/dashboard/main.js

```diff
@@ -65,6 +66,65 @@ window.Dashboard = {
 		Marbles.history.navigate('login'+ redirectPath);
 	},
 
+	__isCertInstalled: function() {
+		var self = this;
+		if (window.location.protocol === "https:") {
+			self.client.ping("controller", "https").catch(function (args) {
+				var xhr = args[1];
+				if (xhr.status === 0) {
+					// We were not able to access the controller due to a network error (ssl, timeout)
+					// In order to understand what's happening, we have to switch to http.
+					self.dispatcher.handleAppEvent({
```

Comment:
I get until here, but `self.dispatcher` is not defined. How can I access the dispatcher?